partialWaveFit/fitResult.cc

```diff
@@ -569,14 +569,17 @@ double
 fitResult::intensityErr(const string& waveNamePattern) const
 {
 	// get amplitudes that correspond to wave name pattern
-	const vector<unsigned int> prodAmpIndices = prodAmpIndicesMatchingPattern(waveNamePattern);
-	const unsigned int         nmbAmps        = prodAmpIndices.size();
-	if (not _covMatrixValid or (nmbAmps == 0)) {
+	const vector<unsigned int> waveIndices    = waveIndicesMatchingPattern(waveNamePattern);
+	const unsigned int         nmbWaves       = waveIndices.size();
+	if (not _covMatrixValid or (nmbWaves == 0)) {
 		return 0.;
 	}
-	if(nmbAmps == 1) {
-		return intensityErr(prodAmpIndices[0]);
+	if(nmbWaves == 1) {
+		return intensityErr(waveIndices[0]);
 	}
+
+	const vector<unsigned int> prodAmpIndices = prodAmpIndicesMatchingPattern(waveNamePattern);
```

Comment:
`waveNamePattern` is now used once as a pattern for a wave name and once for a pattern for production amplitudes (the name `waveNamePattern` obviously was misleadingly used to begin with). When creating the standard spin totals for three pions I observed the necessity to prefix the pattern for the production amplitudes with `V\\d*_` as otherwise wrong patterns would be matched and the intensity error would be calculated wrong. From a performance point of view it would be a nightmare to write something like:
```
vector<unsigned int> prodAmpIndices;
for (unsigned int i = 0; i < nmbWaves; ++i) {
	const vector<unsigned int> prodAmpIndicesWave = prodAmpIndicesForWave(waveIndices[i]);
	prodAmpIndices.insert(prodAmpIndices.end(), prodAmpIndicesWave.begin(), prodAmpIndicesWave.end());
}
```